### ISSUE TITLE
Cannot install 5.2.2 in docker container

### ISSUE THREAD

**Issue by manfreddz** [kamailio/kamailio#1935]
### Description

I'm trying to install Kamailio 5.2.2 from the official rpm-repos in a CentOS6 docker-container. I'm running it on Ubuntu 18.04 in docker 18.09.2. I'm getting errors about `Package does not match intended download`.

#### Reproduction
```
$ docker run -it  centos:6.9 bash
[root@8d05c1d9a3d7 /]# yum install -y wget && \
> cd /etc/yum.repos.d && \
> wget 'http://download.opensuse.org/repositories/home:/kamailio:/v5.2.x-rpms/CentOS_6/home:kamailio:v5.2.x-rpms.repo' && \
> yum install -y kamailio
Loaded plugins: fastestmirror, ovl
Setting up Install Process
base                                                                                      | 3.7 kB     00:00     
base/primary_db                                                                           | 4.7 MB     00:02     
extras                                                                                    | 3.4 kB     00:00     
extras/primary_db                                                                         |  29 kB     00:00     
updates                                                                                   | 3.4 kB     00:00     
updates/primary_db                                                                        | 3.7 MB     00:01     
Resolving Dependencies
--> Running transaction check
---> Package wget.x86_64 0:1.12-10.el6 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

=================================================================================================================
 Package                 Arch                      Version                         Repository               Size
=================================================================================================================
Installing:
 wget                    x86_64                    1.12-10.el6                     base                    484 k

Transaction Summary
=================================================================================================================
Install       1 Package(s)

Total download size: 484 k
Installed size: 1.8 M
Downloading Packages:
wget-1.12-10.el6.x86_64.rpm                                                               | 484 kB     00:00     
warning: rpmts_HdrFromFdno: Header V3 RSA/SHA1 Signature, key ID c105b9de: NOKEY
Retrieving key from file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
Importing GPG key 0xC105B9DE:
 Userid : CentOS-6 Key (CentOS 6 Official Signing Key) <centos-6-key@centos.org>
 Package: centos-release-6-9.el6.12.3.x86_64 (@CentOS/6.9)
 From   : /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6
Running rpm_check_debug
Running Transaction Test
Transaction Test Succeeded
Running Transaction
  Installing : wget-1.12-10.el6.x86_64                                                                       1/1 
install-info: No such file or directory for /usr/share/info/wget.info.gz
  Verifying  : wget-1.12-10.el6.x86_64                                                                       1/1 

Installed:
  wget.x86_64 0:1.12-10.el6                                                                                      

Complete!
--2019-04-23 14:22:48--  http://download.opensuse.org/repositories/home:/kamailio:/v5.2.x-rpms/CentOS_6/home:kamailio:v5.2.x-rpms.repo
Resolving download.opensuse.org... 195.135.221.134, 2620:113:80c0:8::13
Connecting to download.opensuse.org|195.135.221.134|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 308
Saving to: `home:kamailio:v5.2.x-rpms.repo'

100%[=======================================================================>] 308         --.-K/s   in 0s      

2019-04-23 14:22:48 (41.6 MB/s) - `home:kamailio:v5.2.x-rpms.repo' saved [308/308]

Loaded plugins: fastestmirror, ovl
Setting up Install Process
Determining fastest mirrors
 * base: mirror.nsc.liu.se
 * extras: mirror.nsc.liu.se
 * updates: mirror.nsc.liu.se
home_kamailio_v5.2.x-rpms                                                                 | 1.3 kB     00:00     
home_kamailio_v5.2.x-rpms/primary                                                         | 8.9 kB     00:00     
home_kamailio_v5.2.x-rpms                                                                                  47/47
Resolving Dependencies
--> Running transaction check
---> Package kamailio.x86_64 0:5.2.2-4.el6 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

=================================================================================================================
 Package               Arch                Version                  Repository                              Size
=================================================================================================================
Installing:
 kamailio              x86_64              5.2.2-4.el6              home_kamailio_v5.2.x-rpms              5.5 M

Transaction Summary
=================================================================================================================
Install       1 Package(s)

Total download size: 5.5 M
Installed size: 25 M
Downloading Packages:
kamailio-5.2.2-4.el6.x86_64.rpm                                                           | 5.5 MB     00:02     
http://download.opensuse.org/repositories/home%3A/kamailio%3A/v5.2.x-rpms/CentOS_6/x86_64/kamailio-5.2.2-4.el6.x86_64.rpm: [Errno -1] Package does not match intended download. Suggestion: run yum --enablerepo=home_kamailio_v5.2.x-rpms clean metadata
Trying other mirror.


Error Downloading Packages:
  kamailio-5.2.2-4.el6.x86_64: failure: x86_64/kamailio-5.2.2-4.el6.x86_64.rpm from home_kamailio_v5.2.x-rpms: [Errno 256] No more mirrors to try.
```

The error reads `kamailio-5.2.2-4.el6.x86_64.rpm: [Errno -1] Package does not match intended download. Suggestion: run yum --enablerepo=home_kamailio_v5.2.x-rpms clean metadata`. The suggestion doesn't help. 

Have I made a mistake or is the repo broken?

* **Operating System**:

```
$ lsb_release  -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.2 LTS
Release:	18.04
Codename:	bionic
$ uname -a
Linux madell-thinkpad 4.15.0-47-generic #50-Ubuntu SMP Wed Mar 13 10:44:52 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
$ docker version
Client:
 Version:           18.09.2
 API version:       1.39
 Go version:        go1.10.4
 Git commit:        6247962
 Built:             Tue Feb 26 23:52:23 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.09.2
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.4
  Git commit:       6247962
  Built:            Wed Feb 13 00:24:14 2019
  OS/Arch:          linux/amd64
  Experimental:     false
$ docker image inspect centos:6.9
[
    {
        "Id": "sha256:2199b8eb8390197d175b1dd57ed79f53ed92ffa58e23826ada54113262237e56",
        "RepoTags": [
            "centos:6.9"
        ],
        "RepoDigests": [
            "centos@sha256:8c538ac91da5eb1be8f4962cb02f7171043aa782eac22b6046fecc0fd4d006c5"
        ],
        "Parent": "",
        "Comment": "",
        "Created": "2019-03-14T21:22:46.023197937Z",
        "Container": "d3e1bd47b7bc5704e90893c57f35b4b446adc24229e93cec053e5dbddfbd56ce",
        "ContainerConfig": {
            "Hostname": "d3e1bd47b7bc",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
            ],
            "Cmd": [
                "/bin/sh",
                "-c",
                "#(nop) ",
                "CMD [\"/bin/bash\"]"
            ],
            "ArgsEscaped": true,
            "Image": "sha256:2949d2914bc8bcf330ffdea46d30b8798fe9be92c6bfdca14ffb8b3ed4a9afc9",
            "Volumes": null,
            "WorkingDir": "",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": {
                "build-date": "20170406",
                "license": "GPLv2",
                "name": "CentOS Base Image",
                "vendor": "CentOS"
            }
        },
        "DockerVersion": "18.06.1-ce",
        "Author": "",
        "Config": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
            ],
            "Cmd": [
                "/bin/bash"
            ],
            "ArgsEscaped": true,
            "Image": "sha256:2949d2914bc8bcf330ffdea46d30b8798fe9be92c6bfdca14ffb8b3ed4a9afc9",
            "Volumes": null,
            "WorkingDir": "",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": {
                "build-date": "20170406",
                "license": "GPLv2",
                "name": "CentOS Base Image",
                "vendor": "CentOS"
            }
        },
        "Architecture": "amd64",
        "Os": "linux",
        "Size": 194752565,
        "VirtualSize": 194752565,
        "GraphDriver": {
            "Data": null,
            "Name": "aufs"
        },
        "RootFS": {
            "Type": "layers",
            "Layers": [
                "sha256:aaa5621d7c0157cae5916c9cca66dd8fc2fb4bdb74813ed463b73d5b58cccfdf"
            ]
        },
        "Metadata": {
            "LastTagTime": "0001-01-01T00:00:00Z"
        }
    }
]
```

**Comment by sergey-safarov** [comment]
Triggered repo rebuild
You cant try again after 30 min

**Comment by manfreddz** [comment]
Now CentOS6 fails with an error message about %{_rundir}/kamailio being an invlid home directory.

**Comment by sergey-safarov** [comment]
Fixed at d7c117b961b6c5b6d40cec9b81de94780578d5f2
